porch/config/deploy/3-porch-server.yaml

```diff
@@ -42,7 +42,7 @@ spec:
         - name: porch-server
           # Update image to the image of your porch apiserver build.
           image: gcr.io/example-google-project-id/porch-server:latest
-          imagePullPolicy: IfNotPresent
+          imagePullPolicy: Always
```

Comment:
Good catch (or we can probably just drop the field entirely and rely on defaulting)